pySDC/tests/test_projects/test_second_orderSDC/test_convergence.py

```diff
@@ -0,0 +1,177 @@
+import pytest
+
+
+METHODS = ['RKN', 'Velocity_Verlet']
+
+
+def get_sweeper(sweeper_name):
+    """
+    Retrieve a sweeper from a name
+
+    Args:
+        sweeper_name (str):
+
+    Returns:
+        pySDC.Sweeper.RungeKutta: The sweeper
+    """
+    import pySDC.implementations.sweeper_classes.Runge_Kutta_Nystrom as RK
+
+    return eval(f'RK.{sweeper_name}')
+
+
+def penningtrap_params(sweeper_name):
+    import numpy as np
+    from pySDC.implementations.problem_classes.PenningTrap_3D import penningtrap
+
+    # initialize level parameters
+    level_params = dict()
+    level_params['restol'] = 1e-16
+    level_params['dt'] = 0.015625
+
+    # initialize sweeper parameters
+    sweeper_params = dict()
+    sweeper_params['quad_type'] = 'GAUSS'
+    sweeper_params['num_nodes'] = 5
+    sweeper_params['do_coll_update'] = True
+    sweeper_params['initial_guess'] = 'zero'  # 'zero', 'spread'
```

Comment:
Why do you choose zero here? I found that usually spread works much better.

Reply:
That is right, spread works better but to reach minimal convergence order like in the project. I needed to use zero.